shared/messaging/envelope_test.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_UnpackMessage_roundtrip(t *testing.T) {
+func Test_UnpackMessage_Roundtrip(t *testing.T) {
 	someMsg := &DebugMessage{Action: DebugMessageAction_DEBUG_CLEAR_STATE}
 	packedMsg, err := PackMessage(someMsg)
 	require.NoError(t, err)
 
-	unpackedMsg, err := UnpackMessage(packedMsg)
+	unpackedMsg, err := UnpackMessage[*DebugMessage](packedMsg)
 	require.NoError(t, err)
 
-	if !proto.Equal(someMsg, unpackedMsg.(proto.Message)) {
+	if !proto.Equal(someMsg, unpackedMsg) {
```

Comment:
`require.True(t, proto.Equal(someMsg, unpackedMsg))`?

Ditto elsewhere

Reply:
Done, but I can't find the same issue anywhere else 🦮

Reply:
For some reason I thought I saw it in another place. Please ignore and 🦮 on